src/Forms/GridField/GridFieldDetailForm_ItemRequest.php

```diff
@@ -945,4 +963,48 @@ private function getModelName(): string
         }
         return ClassInfo::shortName($this->record);
     }
+
+    /**
+     * Get Pjax response negotiator so form submission mirrors other form submission in the CMS.
+     * See LeftAndMain::getResponseNegotiator()
+     */
+    private function getResponseNegotiator(DBHTMLText $renderedForm): PjaxResponseNegotiator
+    {
+        return new PjaxResponseNegotiator([
+            'Default' => function () use ($renderedForm) {
```

Comment:
Are you sure this is correct? LeftAndMain::getResponseNegotiator() has a 'default' key instead (lowercase)

Reply:
Good catch, fixed